src/main/resources/internal-admin-api.yaml

```diff
@@ -1303,6 +1303,24 @@ paths:
         200:
           description: "Job started"
 
+  /testing-support/mock/refdata/external/v1/organisations/mfa:
```

Comment:
Do we need a ticket to take all testing support endpoints out of `internal-admin-api.yaml`?

Reply:
yeah - maybe that could be part of the swagger tidy up?